msi/release.sh

```diff
@@ -52,7 +135,7 @@ BUILD_EXTRA_WINPATH="$(cd "$SCRIPT_PATH"/.. && pwd -W | tr / \\\\)"
             </Component>
             <Component Directory="BinFolder">
                 <File Source="mingw64\\share\\git\\compat-bash.exe"
-                      Name="sh.exe" />
+		      Name="sh.exe" />
```

Comment:
Okay, so this is a whitespace fix. Maybe you could accumulate all those whitespace fixes into a single commit that goes in the front of the patch series?